client/my-sites/plans/jetpack-plans/product-lightbox/constants.ts

```diff
@@ -1,7 +1,15 @@
-export const PRODUCT_OPTIONS = {
-	Backup: [
-		{ id: 'jetpack_backup_t0_yearly', answerText: '1GB' },
-		{ id: 'jetpack_backup_t1_yearly', answerText: '10GB' },
-		{ id: 'jetpack_backup_t2_yearly', answerText: '1TB(1000GB)' },
-	],
+import {
+	PRODUCT_JETPACK_BACKUP_T2_YEARLY,
+	PRODUCT_JETPACK_BACKUP_T1_YEARLY,
+} from '@automattic/calypso-products';
+import { translate } from 'i18n-calypso';
+
+export const PRODUCT_OPTIONS: Record< string, string > = {
+	[ PRODUCT_JETPACK_BACKUP_T1_YEARLY ]: translate( '10GB' ),
+	[ PRODUCT_JETPACK_BACKUP_T2_YEARLY ]: translate( '1TB(1000GB)' ),
```

Comment:
We should add a space between the 1TB and the parenthesis, as shown here in this Slack comment: p1662725061135259-slack-C03RCHNVC0H

And as a courtesy to our translators we could also add a `comments:` argument to explain that GB is the abbreviation for Gigabyte and TB is the abbreviation for Terabyte. (Although this is just a nitpick suggestion).